src/main/java/frc/robot/Constants.java

```diff
@@ -225,7 +231,15 @@ public static final class ArmConstants {
     public static final double ARM_MAX_OUTPUT = 1.0;
     
     public static final double ARM_ENCODER_OFFSET = 0;
-  
+  }
 
+  public static final class Localizer {
```

Comment:
this section in constants can be deleted

Reply:
This is used in the Localizer Class.